packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettings.tsx

```diff
@@ -30,6 +33,9 @@ export const DatabaseSettings: React.FC<Props> = (props) => {
   );
 
   const isLegacy = database.platform === 'rdbms-legacy';
+  const isDefault = database.platform === 'rdbms-default';
+
+  const suspendClusterCopy = `Suspend the cluster if it's temporarily not used to prevent being billed for it.`;
```

Comment:
This copy doesn't read super well to me.  

Here's what I came up with but we may want to check with UX or a copy/technical writer

```suggestion
  const suspendClusterCopy = `You can suspend a cluster temporarily if it is not being used. You will not be billed for it.`;
```

Reply:
Hi @bnussman-akamai , we're not using "You can" as users know that if the option is available that "they can", and I'm not sure "You will not be billed for it" would be clear that by "it" we mean "the suspended cluster". How about:

Suspend the cluster if you don't use it temporarily to prevent being billed for it. ?

Reply:
@bnussman-akamai Text for the view above was provided by @m-sulik . Updating to the latest provided above:
`Suspend the cluster if you don't use it temporarily to prevent being billed for it.`

Please confirm if this text works.